src/util.cpp

```diff
@@ -173,6 +173,21 @@ std::string LogGetAllString()
     return ret;
 }
 
+std::string LogGetAllEnableCategoriesString()
```

Comment:
This is very, very similar to `LogGetAllString()`. Can't these two be combined or just one of them used?

Also, the check `x.first == NONE` seems superfluous and confusing (in both of the functions).

Reply:
Add to `LogGetAllString` a functionality to show a string containing the string representation of active debug category separated by white spaces.